sysdig/internal/client/v2/cloudauth.go

```diff
@@ -119,22 +123,20 @@ func (client *Client) getCloudauthAccountURL(accountID string, decrypt string) s
 	return fmt.Sprintf(getCloudauthAccountPath, client.config.url, accountID, decrypt)
 }
 
-// local function for protojson based marshal/unmarshal of cloudauthAccount proto
-func (client *Client) marshalProto(data *CloudauthAccountSecure) (io.Reader, error) {
-	payload, err := protojson.Marshal(data)
+// common func for protojson based marshal/unmarshal of any cloudauth proto
+func (client *Client) marshalCloudauthProto(message protoreflect.ProtoMessage) (io.Reader, error) {
```

Comment:
nit: using `protobuf.Message` a mere type alias of `protoreflect.ProtoMessage` is preferred (for some reason).

Reply:
nit: this function doesn't really have anything to with cloudauth now, it's a protojson Marshaller.

Reply:
Yes, since all the cloudauth related resources (account, org and now AccountComponent) require this marshaling/unmarshaling, I refactored the func in a common place and to accept _any_ proto message type. Let me check changing the type to `protobuf.Message`

Reply:
Done.